### PR TITLE
grafana/toolkit: Support js plugins

### PR DESCRIPTION
**Pull request by dprokop**

### CODE REVIEW

File: packages/grafana-toolkit/src/config/jest.plugin.config.ts [hunk 1]
```diff
@@ -22,7 +28,7 @@ export const jestConfig = (baseDir: string = process.cwd()) => {
   }
 
   const shimsFilePath = path.resolve(baseDir, 'config/jest-shim.ts');
-  const setupFilePath = path.resolve(baseDir, 'config/jest-setup.ts');
+  const setupFilePath = path.resolve(baseDir, 'config/jest-setup.js');
```
Comment (dprokop) [inline packages/grafana-toolkit/src/config/jest.plugin.config.ts]: Forgot about that, need to update to choose either js or ts file

File: packages/grafana-toolkit/src/config/jest.plugin.config.ts [hunk 2]
```diff
@@ -42,11 +59,15 @@ export const jestConfig = (baseDir: string = process.cwd()) => {
     setupFiles,
     globals: { 'ts-jest': { isolatedModules: true } },
     coverageReporters: ['json-summary', 'text', 'lcov'],
-    collectCoverageFrom: ['src/**/*.{ts,tsx}', '!**/node_modules/**', '!**/vendor/**'],
+    collectCoverageFrom: ['src/**/*.{ts,tsx}', '!**/node_modulers/**', '!**/vendor/**'],
```
Comment (dprokop) [inline packages/grafana-toolkit/src/config/jest.plugin.config.ts]:
```suggestion
    collectCoverageFrom: ['src/**/*.{ts,tsx}', '!**/node_modules/**', '!**/vendor/**'],
```

**Review by torkelo — APPROVED** [review]
looks good!

File: packages/grafana-toolkit/src/config/webpack.plugin.config.ts [hunk 3]
```diff
@@ -89,6 +89,7 @@ const getCommonPlugins = (options: WebpackConfigurationOptions) => {
         { from: '**/*.svg', to: '.' },
         { from: '**/*.png', to: '.' },
         { from: '**/*.html', to: '.' },
+        { from: 'libs/*', to: '.' },
```
Comment (ryantxu) [inline packages/grafana-toolkit/src/config/webpack.plugin.config.ts]: should we add 'static/*' while we are at it?

**Review by ryantxu — APPROVED** [review]
looks great!  thank you

File: packages/grafana-toolkit/src/config/webpack.plugin.config.ts (hunk 3 above)
Comment (dprokop) [inline packages/grafana-toolkit/src/config/webpack.plugin.config.ts]:
```suggestion
        { from: 'libs/*', to: '.' },
        { from: 'static/*', to: '.' },
```

**Pull request merged**